sdk/servicebus/service-bus/test/serviceBusClient.spec.ts

```diff
@@ -33,7 +33,7 @@ import {
 import { ClientType } from "../src/client";
 import { DispositionType } from "../src/serviceBusMessage";
 import { getEnvVars, isNode } from "./utils/envVarUtils";
-import { loginWithServicePrincipalSecret } from "./utils/aadUtils";
+import { loginWithServicePrincipalSecret } from "@azure/ms-rest-nodeauth";
```

Comment:
It looks like you're depending on the node.js specific library in your test now, and that might be confusing rollup when generating the browser build.

Reply:
Yes, this is it! Matches results from local experiments